rails/README.md

```diff
@@ -4,6 +4,7 @@ A guide for building great Rails apps.
 
 - Use def `self.method`, not the `scope :method`
 - Use methods instead of instance variables in ViewComponent templates
+- Use i18n for all text.
```

Comment:
Do you think we should touch on I18n usage within tests as well?

Reply:
In case we want to treat it differently. Please disregard if we don't.

Reply:
Good idea!

Reply:
My two cents: Within specs, the MD team has abandoned the usage of `i18n` in favor of raw text and grabbing elements from the user's perspective. This is particularly useful when reusing specs between Rails and React as well, apart from improvement in readability and making them more robust (they wouldn't change unless the feature changes)

Reply:
I agree with @Simonpedro here. I find it particularly useful to have the actual text in the test in two scenarios:

- when reading a test without much context 
- when practicing TDD

Reply:
Since this seems to be a divided topic, I'll leave it out of this PR.

Reply:
Just in case it wasn't that clear, @arzezak is also onboard with using raw text in specs. 

We could ask in #engineering to see what are the preferences and open another PR if needed, sounds good!

Reply:
Gotcha, I misunderstood Ariel's comment.

Reply:
Sorry about that. I wasn't stating a preference. I was trying to highlight the fact that we might want to treat tests differently than application code and ask if we should state a preferred approach here, too.

Reply:
Updated!

Reply:
FWIW, as Pedro said, I favor keeping specs as bare and close to the user experience as possible. That is, I'd prefer to have to change a spec when the text we expect to show changes and when we change the mechanism by which we fetch translations (even more so while we migrate from React to Rails).